src/vs/workbench/contrib/chat/browser/chat.ts

```diff
@@ -50,6 +52,19 @@ export async function showEditsView(viewsService: IViewsService): Promise<IChatW
 	return (await viewsService.openView<ChatViewPane>(EditsViewId))?.widget;
 }
 
+export function ensureSideBarChatViewSize(width: number, viewDescriptorService: IViewDescriptorService, layoutService: IWorkbenchLayoutService): void {
```

Comment:
What are you trying to prevent with this, just looking weird when the sidebar was sized to be really narrow?